registrations/tests/test_registration_put.py

```diff
@@ -366,6 +366,8 @@ def test_email_is_not_sent_if_registration_user_access_email_is_not_updated(
         email=user_email,
         is_substitute_user=is_substitute_user,
     )
+    # An email is sent when registration user access is created
+    assert len(mail.outbox) == 1
```

Comment:
In addition to these mail count asserts, I guess it would be a bit more robust to verify that the mail that was sent is indeed the correct one. Isn't there already `assert_invitation_email_is_sent()` for that, or is there some problem with it?

Reply:
The problem was that the desired email content was compered to a wrong email. Solved it by clearing mail.outbox before updating registration user access object.